ci.py

```diff
@@ -190,7 +191,8 @@ def check_deps(self):
             repo_path = '/'.join(target_url.split('/')[-2:])
             target_branch = dep['branch']
             repo = github_api.get_repo(repo_path)
-            head = f'{repo.organization.name}:{target_branch}'
+            org = repo.organization.name if repo.organization else repo.owner.login
+            head = f'{org}:{target_branch}'
```

Comment:
to support non-org repos

Reply:
[here is an example](https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-submodules/detail/PR-2220/2/pipeline#step-17-log-63) where it fails